README.md

```diff
@@ -37,21 +34,93 @@ $ ./spin --help
-[Rust](https://developer.fermyon.com/spin/rust-components/) or [Go](https://developer.fermyon.com/spin/go-components/)
+[Rust](https://developer.fermyon.com/spin/rust-components/), [JavaScript](https://developer.fermyon.com/spin/javascript-components), [Python](https://developer.fermyon.com/spin/python-components), or [Go](https://developer.fermyon.com/spin/go-components/)
 language guides, and the [guide on configuring Spin applications](https://developer.fermyon.com/spin/configuration/).
 
-After you build your application, run it using Spin:
+Below is an example of using the `spin` CLI to create a new Spin Python application, then adding a JavaScript component:
+
+```bash
+# Create a new Spin application based on the Python language template.
+$ spin new http-py hello-python
+# Add a new JavaScript component based on the language template.
+$ spin add http-js goodbye-javascript
+```
+
+Running the `spin add` command will generate the proper configuration for our component and add it to the [`spin.toml` manifest file](https://developer.fermyon.com/spin/manifest-reference). For example, here is the `spin.toml` section for our Python component:
 
+```toml
+[[component]]
+# The ID of the component.
+id = "hello-python"
+# The Wasm module to instantiate and execute when receiving a request.
+source = "hello-python/app.wasm"
+[component.trigger]
+# The route for this component.
+route = "/hello"
+[component.build]
+# The command to execute for this component with `spin build`.
+command = "spin py2wasm app -o app.wasm"
+# The working directory for the component.
+workdir = "hello-python"
 ```
+
+We can now build our application with `spin build`, then run it locally with `spin up`:
+
+```bash
+# Compile all components to Wasm by executing their `build` commands.
+$ spin build
+Executing the build command for component hello-python: spin py2wasm app -o app.wasm
+Executing the build command for component goodbye-javascript: npm run build
+Successfully ran the build command for the Spin components.
+# Run the application locally.
 $ spin up
+Logging component stdio to ".spin/logs/"
+Serving http://127.0.0.1:3000
+Available Routes:
+  hello-python: http://127.0.0.1:3000/hello
+  goodbye-javascript: http://127.0.0.1:3000/goodbye
+```
+
+Once the application is running, we can start testing it by sending requests to its components:
+
+```bash
+# Send a request to the Python component.
+$ curl localhost:3000/hello
+Hello, Python!
+# Send a request to the JavaScript component.
+$ curl localhost:3000/goodbye
+Goodbye, JavaScript!
 ```
 
+When handling a request, Spin will create a new isolated Wasm instance corresponding to the Wasm module for the matching component, execute the handler function, then terminate the instance. Each new request will get a fresh Wasm instance.
+
+## Language Support for Spin Features
+
+The table below shows [feature support](https://developer.fermyon.com/spin/language-support-overview) in each of the language SDKs.
```

Comment:
I worry about how well maintained this will be over time.  Do we need it in the repo read-me?  Could we not link to the docs?

Reply:
I also have my concerns about maintaining this list.  The big benefit I see to keeping this in is the list of features.  We do not have a straight list of features anywhere else and if we list them, we should also set expectation on the language support.  Hopefully we will get to the point where all the features are supported by all SDKs.